src/App.js

```diff
@@ -32,7 +54,9 @@ class App extends Component {
             <TurtleCanvas width={300} height={300} rotation={45}/>
           </div>
 
-          <Button id="gcode" bsStyle="primary">Generate GCode</Button>
+          <div id="output">
+            <GCodeGenerator vertices={this.state.vertices}/>
```

Comment:
I want to pass `this.state.vertices` into Shapes and Transforms, or maybe I'll need to just send it into Transforms and have a separate list of vertices come from Shapes and go to Transforms. But they will be responsible for generating the vertices.

I also want to pass the vertices to the canvas and that's how we'll get a live preview.

If/When we have other modes for generating vertices, like a spirograph, or turtle output, they would also create the vertices. They would end up in the canvas and ultimately in the gcode.